inst/rmarkdown/templates/isba/skeleton/README.md

```diff
@@ -0,0 +1,36 @@
+## About
```

Comment:
Same for this file - does it really need to be copied into the user directory ? 

I would put it one level above, or in the resource folder

Reply:
Removed